packages/common/stateInfo.ts

```diff
@@ -59,7 +60,7 @@ export const eligibleSignatureType = (state: ImplementedState) => {
 
 export interface BaseInfo extends Locale {
   state: ImplementedState
-  name: string
+  name: Name
```

Comment:
For backward compatibility, let's keep `name` as a string (the full name) and have another field `nameParts`.  In general, we shouldn't break backward compatibility unless we have to when it comes to data storage.